dub.json

```diff
@@ -12,12 +12,18 @@
 			"libs-windows-x86_64": [
 				"python27"
 			],
+			"libs-windows-x86_mscoff": [
+				"python27"
+			],
 			"libs-posix": [
 				"python2.7"
 			],
 			"lflags-windows-x86_64": [
 				"/LIBPATH:C:\\Users\\$USERNAME\\AppData\\Local\\Programs\\Python\\Python27\\libs"
 			],
+			"lflags-windows-x86_mscoff": [
+				"/LIBPATH:C:\\Users\\$USERNAME\\AppData\\Local\\Programs\\Python\\Python27-32\\libs"
```

Comment:
haven't used dub in ages, does it provide a way to override configurations like this? I expect python doesn't always live there

Reply:
It doesn't always live there, but that's the default for the installer (or at least it is these days). I can't see how else to do this other than use an environment variable instead, but then that variable would always have to be set by the user, manually.